misaligned.cpp

```diff
@@ -1,21 +1,29 @@
 #include <iostream>
 #include <assert.h>
 
-int printColorMap() {
+std::string getMajorColor(int x) {
     const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+    return majorColor[x];
+}
+
+std::string getMinorColor(int x) {
     const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+    return minorColor[x];
+}
+
+void printColorMap() {
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+            std::cout << i * 5 + j << " | " << getMajorColor(i) << " | " << getMinorColor(i) << "\n";
         }
     }
-    return i * j;
 }
 
 int main() {
-    int result = printColorMap();
-    assert(result == 25);
+    assert("0 | " + getMajorColor(0) + " | " + getMinorColor(0) == "0 | White | Blue");
+    assert("1 | " + getMajorColor(1) + " | " + getMinorColor(1) == "1 | White | Orange");
+    assert("2 | " + getMajorColor(2) + " | " + getMinorColor(2) == "2 | White | Green");
     std::cout << "All is well (maybe!)\n";

```

Comment:
The number 5 used in the pair number calculation would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.